examples/performance-subscribe/src/PerformanceTestApp.cpp

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "PerformanceTestApp.h"
+#include "sdk/IPubSubClient.h"
+#include "sdk/Logger.h"
+#include "sdk/vdb/IVehicleDataBrokerClient.h"
+
+#include <fmt/chrono.h>
+#include <fmt/core.h>
+
+#include <chrono>
+#include <utility>
+
+namespace example {
+
+namespace {
+std::string getValueRepresentation(const velocitas::DataPointValue& value) {
+    if (!value.isValid()) {
+        return toString(value.getFailure());
+    }
+    return value.getValueAsString();
+}
+
+template <typename TIME_BASE> TIME_BASE getTimestamp() {
```

Comment:
```suggestion
template <typename TTimeBase> TTimeBase getTimestamp() {
```

To align with remaining template parameters, we should rename it. We are still missing a reference to a code styleguide in our CONTRIBUTING.md